src/histolab/slide.py

```diff
@@ -274,113 +277,105 @@ def slides_dimensions(self):
         ]
 
     @property
-    def slides_stats(self):
-        """Retrieve statistic/graphs of wsi files contained in the dataset"""
-        t = Time()
+    def _wsi_dimensions_list(self) -> List[tuple]:
```

Comment:
It's not clear to me why we should have both `_wsi_dimensions_list ` and `_slides_dimensions` properties. They represent the same thing (if I'm not missing anything). I would keep `_slides_dimensions` (more structured).

Moreover, we should decide between "slide" or "wsi" naming. They represent the same thing but here it seems not